app/views/facility_orders/_merge_order_form.html.haml

```diff
@@ -3,6 +3,12 @@
     = simple_form_for [current_facility, @add_to_order_form], url: facility_order_path(current_facility, @order), method: :put, html: { class: "js--edit-order" } do |f|
       = f.input_field :quantity, class: "js--edit-order__quantity", style: "width: 3em;"
 
+      - unless Rails.env.production? || !can?(:available_for_cross_core_ordering, Product)
```

Comment:
How about putting this on a separate line, above the quanatity/product inputs?  And let's give this and the product dropdown a label as well